interfaces/builtin/unity8_calendar.go

```diff
@@ -0,0 +1,130 @@
+# Usage: reserved
+
+# DBus accesses
+#include <abstractions/dbus-session-strict>
+dbus (bind)
+	bus=session
+	name="org.gnome.evolution.dataserver.Calendar7",
+dbus (bind)
+	bus=session
+	name=org.gnome.evolution.dataserver.Subprocess.Backend.*,
+
+########################
+# Calendar
+########################
+dbus (receive, send)
+	bus=session
+	path=/org/gnome/evolution/dataserver/CalendarFactory,
+dbus (receive, send)
+	bus=session
+	path=/org/gnome/evolution/dataserver/CalendarFactory
+	interface=org.gnome.evolution.dataserver.CalendarFactory,
+dbus (receive, send)
+	bus=session
+	path=/org/gnome/evolution/dataserver/CalendarFactory
+	interface=org.freedesktop.DBus.*,
+
+dbus (receive, send)
+	bus=session
+	path=/org/gnome/evolution/dataserver/CalendarView/**,
+dbus (receive, send)
+	bus=session
+	path=/org/gnome/evolution/dataserver/CalendarView/**
+	interface=org.gnome.evolution.dataserver.CalendarView,
+dbus (receive, send)
+	bus=session
+	path=/org/gnome/evolution/dataserver/CalendarView/**
+	interface=org.freedesktop.DBus.*,
+
+dbus (receive, send)
+	bus=session
+	path=/org/gnome/evolution/dataserver/Subprocess/Backend/Calendar/**
+	interface=org.gnome.evolution.dataserver.Subprocess.Backend,
+dbus (receive, send)
+	bus=session
+	path=/org/gnome/evolution/dataserver/Subprocess/Backend/Calendar/**
+	interface=org.freedesktop.DBus.*,
+
+dbus (receive, send)
+	bus=session
+	path=/org/gnome/evolution/dataserver/Subprocess/**
+	interface=org.gnome.evolution.dataserver.Calendar,
+dbus (receive, send)
+	bus=session
+	path=/org/gnome/evolution/dataserver/Subprocess/**
+	interface=org.freedesktop.DBus.*,
+`
+
+const unity8CalendarConnectedPlugAppArmor = `
+# Description: Can access the calendar. This policy group is reserved for
+#  vetted applications only in this version of the policy. Once LP: #1227824
+#  is fixed, this can be moved out of reserved status.
+# Usage: reserved
+
+dbus (receive, send)
+     bus=session
+     path=/org/gnome/evolution/dataserver/CalendarFactory
+     peer=(label=unconfined),
+dbus (receive, send)
+     bus=session
+     path=/org/gnome/evolution/dataserver/Subprocess/**
+     peer=(label=unconfined),
+dbus (receive, send)
+     bus=session
+     path=/org/gnome/evolution/dataserver/CalendarView/**
+     peer=(label=unconfined),
+
+# LP: #1319546. Apps shouldn't talk directly to sync-monitor, but allow it for
+# now for trusted apps until sync-monitor is integrated with push
+# notifications.
+dbus (receive, send)
+     bus=session
+     path=/synchronizer{,/**}
+     peer=(label=unconfined),
+`
+
+const unity8CalendarPermanentSlotDBus = `
+	<allow own="org.gnome.evolution.dataserver.Calendar7"/>
+	<allow send_destination="org.gnome.evolution.dataserver.Calendar7"/>
+	<allow send_interface="org.gnome.evolution.dataserver.Calendar"/>
+	<allow send_interface="org.gnome.evolution.dataserver.CalendarView"/>
+	<allow send_interface="org.gnome.evolution.dataserver.CalendarFactory"/>
+`
```

Comment:
This is problematic-- the dbus interface is meant for system bus policy, but eds is a session service. I'm not sure how to advise here since snappy session services have not been designed yet....